ios-training/NewViewController.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  NewViewController.swift
+//  ios-training
+//
+//  Created by Yusuke Mori on 2023/01/26.
+//
+
+import UIKit
+
+class NewViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    // Viewが表示された時にViewControllerへ画面遷移
+    override func viewDidAppear(_ animated: Bool) {
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let nextViewController = storyboard.instantiateInitialViewController() as! ViewController
```

Comment:
[nits]
強制アンラップはクラッシュの原因になるので極力使用しないようにします。
クラッシュするとユーザーの離脱につながるので、サービス運営においては極力避けるようにします。

```suggestion
        guard let nextViewController = storyboard.instantiateInitialViewController() as? ViewController else { return }
```

Reply:
了解しました。
修正します。